doc/changes/latest.inc

```diff
@@ -23,6 +23,8 @@ Enhancements
 
 - Speed up :func:`mne.inverse_sparse.tf_mixed_norm` using STFT/ISTFT linearity (:gh:`8697` by `Eric Larson`_)
 
+- `~mne.Report.parse_folder` now processes supported non-FIFF files by default, too (:gh:`8744` by `Richard Höchenberger`_)
```

Comment:
The `~` will make it show up just as `parse_folder` which will be less easy to read for non-MNE-experts than the full qualified name (or at least `Report.parse_folder`)
```suggestion
- `mne.Report.parse_folder` now processes supported non-FIFF files by default, too (:gh:`8744` by `Richard Höchenberger`_)
```